docs/API/REST-V2-Usage.md

```diff
@@ -52,1784 +54,24 @@ A returned object will always contain its complete list of fields. Any
-
-Get User Permissions
-
-    GET /users/<user_id>/permissions
-
-**Parameters**
-
-| Name | Type    | Description                                                      |
-|------|---------|------------------------------------------------------------------|
-| all  | Boolean | Optional. If included, we will not filter out basic permissions. |
-
-
-Retrieves all permissions for the user (ignoring basic permissions that
-are given to all users, unless all=true is included). The format of the
-response is :
-```
-[ {
- "type": "project",
- "permissions": {
-    "project1": { "project_tasks": 30, "project_patches": 10 },
-    "project2": { "project_tasks": 10, "project_patches": 10 } }
- } 
- {
- "type": "distro",
- "permissions": { "distro1": {"distro_settings": 10 } } 
-  } ]
-```
-
--   type - the type of resources for which the listed permissions apply.
-    Will be "project", "distro", or "superuser"
--   permissions - an object whose keys are the resources for which the
-    user has permissions. Note that these objects will often have many
-    keys, since logged-in users have basic permissions to every project
-    and distro. The values in the keys are objects representing the
-    permissions that the user has for that resource, identical to the
-    format of the permissions field in the POST
-    /users/\<user_id\>/permissions API.
-
-Get All User Permissions For Resource
-
-    GET /users/permissions  
-
-Retrieves all users with permissions for the resource, and their highest permissions, and returns this as a mapping. This ignores basic permissions that are given to all users.  
-
-The format of the body is: 
-
-    {
-      "resource_type": "project",
-        "resources": ["project1", "project2"],
-        "permissions": {
-             "project_tasks: 30,
-                  "project_patches": 10
-                    }
-                    }"
-        }
-    }
-
-##### Delete User Permissions ``````````\`
-
-    DELETE /users/<user_id>/permissions
-
-Deletes all permissions of a given type for a user by deleting their
-roles of that type for that resource ID. This ignores the Basic
-Project/Distro Access that is given to all MongoDB employees.
-
-Note that usage of this endpoint requires that the requesting user have
-security to modify roles. The format of the body is: :
-
-    {
-      "resource_type": "project",
-      "resource_id": "project_id", 
-    }
-
--   resource_type - the type of resources for which to delete
-    permissions. Must be one of "project", "distro", "superuser",
-    or "all". "all" will revoke all permissions for the user.
--   resource_id - the resource ID for which to delete permissions.
-    Required unless deleting all permissions.
-
-##### Get Users for Role
+NOTE: These roles are not part of the OpenAPI spec, since they are part of another package.
```

Comment:
I think this note should go higher so it doesn't seem applicable only to this specific route. Also nit, "these routes"

Reply:
Also the offboard_user route could be generated.

Reply:
Removed note since they're on the same page now. And generated offboard_user.